src/Tribe/Attendee_Registration/Template.php

```diff
@@ -20,34 +20,37 @@ class Tribe__Tickets__Attendee_Registration__Template extends Tribe__Templates {
 	public function hook() {
 
 		// Spoof the context
-		add_filter( 'the_posts', array( $this, 'setup_context' ), -10 );
+		add_filter( 'the_posts', [ $this, 'setup_context' ], -10 );
 
 		// Set and remove the required body classes
-		add_action( 'wp', array( $this, 'set_body_classes' ) );
+		add_action( 'wp', [ $this, 'set_body_classes' ] );
 
 		/*
 		 * Choose the theme template to use. It has to have a higher priority than the
 		 * TEC filters (at 10) to ensure they do not usurp our rewrite here.
 		 */
-		add_filter( 'template_include', array( $this, 'set_page_template' ), 15 );
+		add_filter( 'template_include', [ $this, 'set_page_template' ], 15 );
+
+		add_action( 'tribe_events_editor_assets_should_enqueue_frontend', [ $this, 'should_enqueue_frontend' ] );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: f189d6803869805ea0ff9e6d580e610a</sub>